src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpBuild.cs

```diff
@@ -35,5 +37,25 @@ static void Main(string[] args)
             // TODO: Validate build works as expected
         }
 
+        [Fact]
+        public void BuildWithCommandLine()
+        {
+            VisualStudio.Instance.SolutionExplorer.SaveAll();
+
+            var pathToDevenv = Path.Combine(VisualStudio.Instance.InstallationPath, @"Common7\IDE\devenv.exe");
+            var pathToSolution = VisualStudio.Instance.SolutionExplorer.SolutionFileFullPath;
+            var logFileName = pathToSolution + ".log";
+
+            File.Delete(logFileName);
```

Comment:
@tmeschter: `File.Delete()` is a no-op if the file doesn't exist. Yes, any time you see somebody doing `if (File.Exists(p)) File.Delete(p);` it's an absolute waste of keystrokes (especially because @jaredpar  would point out it's a race anyways.)